index.bs

```diff
@@ -355,6 +355,73 @@ User agents may use a [=new information about the user's intent|number of criter
 to grant access to the [=sensor readings|readings=].
 Access may be granted without prompting the user.
 
+<h3 id="mitigation-strategies-case-per-case">Mitigation strategies applied on a case per case basis</h3>
+
+Each [=sensor type=] will need to be assessed individually
+taking in account the use cases it enables
+and its particular threat profile.
+While some of the below mitigation strategies
+are effective for certain sensors,
+they might also hinder or altogehter prevent certain use cases.
+
+
+<h4 id="limit-max-freqency" dfn>Limit maximum polling frequency</h4>
+
+User agents may mitigate certain threats by
+limiting the maximum polling frequency.
+What maximum polling frequency to choose depends on the [=sensor type=],
+the kind of threats the user agent is trying to protect against,
+the resources of the attacker, etc.
+
+Limiting the maximum polling frequency prevents use cases
+which rely on low latency or high data density.
+
+
+<h4 id="limit-number-of-delivered-readings" dfn>Limit number of delivered readings</h4>
+
+An alternative to [=Limit maximum polling frequency|limiting the maximum polling frequency=] is to
+limit the number of [=sensor readings=] delivered to the author,
+regardless of what frequency the sensor is polled at.
+This allows use cases which have low latency requirement
+to increase polling frequency
+without increasing the amount of data provided.
+
+Discarding intermediary readings prevents certain use cases,
+such as those relying on certain kinds of filters.
+
+
+<h4 id="reducing-accuracy" dfn>Reducing accuracy</h4>
+
+Reducing the accuracy of [=sensor readings=]
+or sensor reading timestamps
+might also help mitigate certain threats,
+so user agents should not provide
+unnecessarily verbose readouts of sensors data.
+
+However, certain use cases require highly accurate readings,
+especially when operations carried out on the readings,
+or time deltas calculated from the timestamps,
+increase innacuracies exponentially.
+
+
+<h4 id="fuzzing" dfn>Fuzzing sensor reading and/or their timestamps</h4>
```

Comment:
wrt to random bias not being effective, do you have a paper we could point to that would indicate this? I think it's useful to include mitigation strategies that don't work and mark them as such, no?

iirc @maryammjd tends to think the notion of "safe limits" fluctuates and depends on attacker's resources (which tend to increase as HW becomes cheaper and tech capabilities improve), so I'm not certain we can actually fix those or that we don't risk having to reduce it in the future to the point it starts blocking some use cases.